dev/cnf/oss_dependencies.maven

```diff
@@ -304,3 +304,6 @@ org.springframework:spring-webmvc:4.3.13.RELEASE
 xalan:xalan:2.7.1
 xerces:xercesImpl:2.11.0
 commons-collections:commons-collections:3.2.2
+org.reactivestreams:reactive-streams:1.0.2
+org.reactivestreams:reactive-streams-tck:1.0.2
+#org.eclipse.microprofile.reactive.streams:microprofile-reactive-streams-operators:1.0
```

Comment:
suggest removing this if it isn't needed ... leaving it commented out is rather equivocal in a file that should be definitive.

Reply:
Done.